charts/controller/values.yaml

```diff
@@ -49,6 +49,21 @@ global:
   host_port: 5555
   # Prefix for the imagepull secret created when using private registry
   secret_prefix: "private-registry"
+  # Clusters with large nodes (24 CPU+) may need to lower or tune the number of GUNICORN_WORKERS.
+  #
+  # Default value is based on your number of CPUs:
+  # - (os.cpu_count() or 4) * 4 + 1
+  # This setting probably should not be higher than max_connections which defaults to 100
+  # gunicorn_workers: 13
+  # If there are enough gunicorn workers to use up max_connections, you may also want to reduce CONN_MAX_AGE
+  # so health checks time out their connection faster and don't use all available connections
+  #
+  # Valid values are a number of seconds, or 0 to disable persistent connections.
+  # Setting the value to "None" will ensure that connections are never timed out.
+  # - 0 
+  # - 600
+  # - "None"
```

Comment:
What is the default value of `conn_max_age` when no value is set in the chart? Sounds like it will be "None" by default... Is that the correct behaviour we want?